org.openhab.binding.zigbee/README.md

```diff
@@ -183,6 +183,16 @@ If the USB dongle is not recognized, it might be necessary to make the dongle's
   ```
 - Plug in the dongle. It should now be recognized properly as ttyUSBx.
 
+##### EmberZNet and EZSP Protocol Version
+
+Silicon Labs (a.k.a. Silabs) Zigbee stack is called "EmberZNet PRO" (formerly "Ember Zigbee") and provides "EmberZNet Serial Protocol" (EZSP) as the protocol for a host application to interact with. openHAB is currenty support EZSP v5, v6, v7, and v8 which covers most if not all available Zigbee dongle based on Silabs Zigbee chips.
+
+Silicon Labs does not currently have a consolidated list of changes by EmberZNet SDK or EZSP protocol version. The EZSP additions, changes and deletions have only ever been listed in the "Zigbee EmberZNet Release Notes" (EmberZNet SDK) under the "New items" section as well as the Silabs UG100 EZSP Reference Guide included with each EmberZNet SDK release.
```

Comment:
Suggest to remove this as it's really not relevant to the user.

Reply:
Changed it moving the small part I still think is relative from and removing the rest.